pkg/controllers/machineinventoryselector/bootstrap.go

```diff
@@ -85,22 +107,24 @@ func (h *handler) bootstrapReadyHandler(obj *v1beta1.MachineInventorySelector, s
 	return status, nil
 }
 
-// getStopElementalAgentLocalPlan returns the local plan to stop elemental-system-agent service in json format
-func getStopElementalAgentLocalPlan() ([]byte, error) {
+// getStopElementalAgentPlan returns the local plan to stop elemental-system-agent service in json format
+func getStopElementalAgentPlan() []byte {
 	stopElementalAgent := applyinator.Plan{
 		OneTimeInstructions: []applyinator.OneTimeInstruction{
 			{
 				CommonInstruction: applyinator.CommonInstruction{
 					Command: "systemctl",
 					Args: []string{
+						"--no-block",
 						"stop",
 						"elemental-system-agent",
 					},
 				},
 			},
 		},
 	}
-	return json.Marshal(stopElementalAgent)
+	plan, _ := json.Marshal(stopElementalAgent)
```

Comment:
should check here for error? otherwise we may be returning empty plan

Reply:
Yes we can, however this is an static value. That's why I omitted it here.